lib/twilio-ruby/framework/version.rb

```diff
@@ -127,7 +127,7 @@ def read_limits(limit = nil, page_size = nil)
         }
       end
 
-      def page(method, uri, params = {}, data = {}, headers = {}, auth = nil, timeout = nil)
+      def page(method, uri, params = {}, data = {}, headers: {}, auth: nil, timeout: nil)
```

Comment:
Just for clarity, this is backwards-compatible in Ruby, correct?

Reply:
Ugh, no, it's not. Fixing.